### PR TITLE
Refactor spaceship

### PR DESCRIPTION
**Pull request by JessickaMoya**

### CODE REVIEW

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Spaceship.java [hunk 1]
```diff
@@ -15,4 +21,37 @@ public Spaceship(final int pX, final int pY, final int llimit, final int ulimit)
...
+        if (numbersLife > 0) {
+            this.numbersLife -= 1;
+        }
+        if (numbersLife == 0) {
+            System.out.print("Game Over");
+        }
+    }
+
+    /**
+     * @return el how many lives spaceship has
```
Comment (Raymundogs) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Spaceship.java:52]: return "el"? you should remove that word

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Spaceship.java [hunk 2]
```diff
@@ -15,4 +21,37 @@ public Spaceship(final int pX, final int pY, final int llimit, final int ulimit)
...
+     * @return a score value ofSpaceship.
+     */
+    public int getScore() {
+        return this.score;
+    }
+
+    /**
+    * @return Ask if spacehip has more than 3 lives
+    */
+    public void isStillAlive() {
```
Comment (Raymundogs) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Spaceship.java:42]: Wouldn't it be better if this method is applied in spaceElement? considering that an alien can also die.

**Review by Raymundogs — APPROVED** [review]

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Bullet.java [hunk 3]
```diff
@@ -110,6 +110,7 @@ public boolean isCollapsedBulletWithSpaceship(final Spaceship spaceship) {
         if (this.getPositionX() == spaceship.getPosX() && this.positionInitialY == spaceship.getPosY()) {
             System.out.println("The bullet has impact Spaceship");
```
Comment (JorgeCaceres-Jala) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Bullet.java:111]: Maybe you may change this to "The bullet has impacted the Spaceship"

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Interface.java [hunk 4]
```diff
@@ -19,6 +19,7 @@
     private static final int SCALE_WIDTH = Space.DEFAULT_WIDTH * 5;
     private static final int SCALE_HEIGHT = Space.DEFAULT_HEIGHT * 15;
     private final Spaceship spaceship = new Spaceship((SCALE_WIDTH - SIZE_IMAGE) / 2, SCALE_HEIGHT - SIZE_IMAGE, 0, SCALE_WIDTH - 23);
+    private final GameOver gameOver = new GameOver();
 
     public Interface() {
```
Comment (JorgeCaceres-Jala) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Interface.java:24]: This class must change it's name to Gameboard

**Review by JorgeCaceres-Jala — APPROVED** [review]

**Pull request merged**